MANIFEST.in

```diff
@@ -1,6 +1,11 @@
 include *.txt
 include README.md
 include COPYING
+include pyproject.toml
+include *.release
+include Changelog
+include *.yml
+include .coveragerc
```

Comment:
I would argue that .yml and coverage are CI and CI result, so they both should be out of the sdist too. (See https://github.com/Unidata/cftime/pull/157/files#diff-380c6a8ebbbce17d55d50ef17d3cf906R12)